### PR TITLE
MANDA-000 update node

### PR DESCRIPTION
**Pull request by bluemagma612**
- uses Debian "Buster" with slim as base image

### CODE REVIEW

File: chrome-headless/Dockerfile [hunk 1]
```diff
@@ -1,130 +1,13 @@
...
-    unzip && \
-    wget https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-${SONAR_SCANNER_VERSION}-linux.zip && \
-    unzip sonar-scanner-cli-${SONAR_SCANNER_VERSION}-linux -d /usr/local/share/ && \
-    chown -R node: /usr/local/share/sonar-scanner-${SONAR_SCANNER_VERSION}-linux
-
-ENV SONAR_RUNNER_HOME "/usr/local/share/sonar-scanner-${SONAR_SCANNER_VERSION}-linux"
-ENV PATH="${SONAR_RUNNER_HOME}/bin:${PATH}"
-
-# Install latest chrome dev package. following example: https://github.com/ebidel/lighthouse-ci/blob/master/builder/Dockerfile
-RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
```
Comment (kaltepeter) [inline chrome-headless/Dockerfile]: I would like to keep chrome driver

File: chrome-headless/Dockerfile [hunk 2]
```diff
@@ -1,130 +1,13 @@
...
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    ca-certificates \
-    curl \
-    netbase \
-    wget \
-    vim \
-    openssh-client \
-    git \
-    python3 \
-    jq \
```
Comment (kaltepeter) [inline chrome-headless/Dockerfile]: I need to keep these extra tools for ci processes that use them

Comment (BobGarzon) [reply chrome-headless/Dockerfile]: most of these come in the base node image so they don't need to be explicitly installed

**Comment by BobGarzon** [comment]
> Looking good. I need to keep the extra tools and chrome driver to keep this fully useful.

why not keep this light then add things in when they are needed or make a different image for other use cases

**Pull request merged**